test/testcafe/spec/AuthenticatorYubikeyView_spec.js

```diff
@@ -35,24 +35,24 @@ async function setup(t) {
   return pageObject;
 }
 
-fixture('Enroll YubiKey Authenticator');
+fixture('Enroll YubiKey Authenticator').meta('v3', true);
 test
   .requestHooks(logger, enrollMock)('enroll with YubiKey authenticator', async t => {
     const pageObject = await setup(t);
 
+    await t.expect(pageObject.getFormTitle()).eql('Set up YubiKey');
+    await t.expect(pageObject.getFormSubtitle()).eql('Insert the YubiKey into a USB port and tap it to generate a verification code.');
+
     await checkConsoleMessages({
       controller: 'enroll-yubikey',
       formName: 'enroll-authenticator',
       authenticatorKey:'yubikey_token',
       methodType: 'otp'
     });
```

Comment:
moved the checkConsoleMessages method lower to wait for the form to fully load

Reply:
Is this happening because the `afterRender` was triggering prematurely? That should be fixed in the latest on `0.0`. If it's ported to `0.1` the order shouldn't need to be changed in this spec.